app/src/main/java/com/mlinde/marvelcomicsapp/details/DetailsFragment.kt

```diff
@@ -54,7 +56,6 @@ class DetailsFragment : Fragment() {
                 binding.bottomSheetDescription.text = "Sorry, no description was given :("
```

Comment:
If you are using ` binding.apply {` then you don't need to use `binding. bottomSheetDescription`. You can just call `bottomSheetDescription` without `binding` word.